src/plugins/sections.ts

```diff
@@ -78,40 +77,44 @@ export default () => {
     filterTransaction: (tr) => {
       return preventGraphicalAbstractTitleEdit(tr)
     },
-    appendTransaction: (transactions, oldState, newState) => {
-      const positionsToInsert: number[] = []
-
-      const tr = newState.tr
-
-      // if (!transactions.some(tr => tr.docChanged)) return null
-
-      newState.doc.descendants((node, pos) => {
-        if (!isSectionNode(node)) {
-          return false
-        }
-
-        // add a paragraph to sections with only titles
-        if (node.childCount === 1) {
-          const childNode = node.child(0)
-
-          if (childNode.type === childNode.type.schema.nodes.section_title) {
-            positionsToInsert.push(pos + node.nodeSize - 1)
-          }
-        }
-      })
-
-      // return the transaction if something changed
-      if (positionsToInsert.length) {
-        // execute the inserts in reverse order so the positions don't change
-        positionsToInsert.reverse()
-
-        for (const pos of positionsToInsert) {
-          const paragraph = newState.schema.nodes.paragraph.create()
-          tr.insert(pos, paragraph)
-        }
-        tr.setMeta('origin', 'sections')
-        return tr
-      }
-    },
+    /*
+      This is commented because after recent major dependencies update it somehow doesn't work well with the new track changes: RangeError due to paragraph adding into the title.
+      It is also a bit fuzzy how it will affect the history.
```

Comment:
I can't reproduce this issue, but I can't see the new paragraph in the history tab!

Reply:
you need to have a manuscript locally that has a section where section_title is the only content inside section. And you need to have the track changes on.